docs/performance-tuning.asciidoc

```diff
@@ -0,0 +1,52 @@
+[[performance-tuning]]
+== Performance Tuning
+
+There are different ways to optimize/tune the performance of the RUM agent. 
+Which options to adjust depends on whether you are optimizing for speed, memory
+usage, bandwidth or storage.
+
+
+[float]
+[[performance-sampling]]
+=== Sampling
+
+The first knob to reach for when tuning the performance of the agent is <<transaction-sample-rate,`transactionSampleRate`>>.
+Adjusting the sampling rate controls what percent of requests are traced.
+By default, the sample rate is set at `1.0`, meaning _all_ requests are traced
+and sent to the APM server. 
```

Comment:
We probably should explain what exactly is the difference between sample and unsampled transactions, e.g. that unsampled transactions will still be reported but without spans, etc.

Reply:
Maybe something like: The Agent will still record the overall time and result of unsampled transactions,
but not context information, labels, or spans.